packages/utils/pack-up/src/node/tasks/vite/build.ts

```diff
@@ -0,0 +1,73 @@
+import chalk from 'chalk';
+import path from 'path';
+import { Observable } from 'rxjs';
+import { build } from 'vite';
+
+import { resolveViteConfig } from './config';
+import { ViteBaseTask } from './types';
+
+import type { TaskHandler } from '../index';
+
+interface ViteBuildTask extends ViteBaseTask {
+  type: 'build:js';
+}
+
+const viteBuildTask: TaskHandler<ViteBuildTask> = {
+  print(ctx, task) {
+    const targetLines = [
+      '   target:',
+      ...ctx.targets[task.runtime].map((t) => chalk.cyan(`    - ${t}`)),
+    ];
+    const entries = [
+      '   entries:',
+      ...task.entries.map((entry) =>
+        [
+          `    – `,
+          chalk.green(`${path.join(ctx.pkg.name, entry.path)}: `),
+          `${chalk.cyan(entry.entry)} ${chalk.gray('→')} ${chalk.cyan(task.output)}`,
+        ].join('')
+      ),
+    ];
+
+    ctx.logger.log(
+      [`Building javascript files:`, `  format: ${task.format}`, ...targetLines, ...entries].join(
+        '\n'
```

Comment:
```suggestion
        os.EOL
```
Same as other PR (and everywhere)

Reply:
yes ✅